test/fixtures/publishers.yml

```diff
@@ -16,6 +16,22 @@ verified:
   verification_token: "e3d801860b2185931ae4dbec06f7119712f928eaf39f0fe0c5a335e42b7eb1c6"
   verified: true
 
+completed:
```

Comment:
Add a comment describing that this is a publisher that has has an email sent with an authentication token